AlphaGo/training/supervised_policy_trainer.py

```diff
@@ -33,9 +33,9 @@ def shuffled_hdf5_batch_generator(state_dataset, action_dataset,
             # get state from dataset and transform it.
```

Comment:
Need to update transform from `np.random.choice` to `transforms[data_idx[1]]`. __But__ then you should also create a canonical ordering of the transformations; don't assume that `BOARD_TRANSFORMATIONS.keys()` will always be in the same order. We might need a new constant

```python
TRANSFORMATION_INDICES = {
'noop': 0,
'rot90': 1,
'rot180': 2,
...
}
```

which would then need to be used both in the function that loads indices from files and in the function that creates and saves them.